Grakn.java

```diff
@@ -114,8 +114,8 @@ public int id() {
 
         enum Type {
             READ(0),
-            READ_REPLICA(2),
-            WRITE(1);
+            WRITE(1),
+            READ_SECONDARY(2);
```

Comment:
The reasoning to rename it to `READ_SECONDARY` is because it allows you to read from *secondary* replicas, whereas `READ` and `WRITE` reads to the *primary* replica.

Reply:
We agreed in a verbal discussion with Haikal to get rid of the `READ_SECONDARY` transaction type and replace it with a new `Option`.

Reply:
Fixed in https://github.com/graknlabs/client-java/pull/243/commits/fec34cc75e0a53bb8180851d162e01023ce73251, https://github.com/graknlabs/client-java/pull/243/commits/62c43681b78726a34566307317bcae39663475db, https://github.com/graknlabs/client-java/pull/243/commits/8f31ae68c44d647e678f8f364bd283958d931e3b, and https://github.com/graknlabs/client-java/pull/243/commits/b8d3aa5d2ccb64f79e1bed18a9bc0940f05f4eec (I don't know why it took me 4 commits to do so :D )